src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/test_webapp_commands.py

```diff
@@ -116,8 +116,8 @@ def test_win_webapp_quick_create_cd(self, resource_group):
 
     @ResourceGroupPreparer(location='japaneast')
     def test_linux_webapp_quick_create(self, resource_group):
-        webapp_name = 'webapp-quick-linux'
-        plan = 'plan-quick-linux'
+        webapp_name = 'webapp-quick-linux4'
+        plan = 'plan-quick-linux4'
```

Comment:
The name should be created randomly, otherwise, the test can't be run in the live environment in parallel and repeatedly.
Doc: https://github.com/Azure/azure-cli/blob/master/doc/scenario_base_tests.md#sample-6-random-name-and-name-mapping